packages/core/src/Dialog/Title/Title.tsx

```diff
@@ -48,7 +48,7 @@ export const HvDialogTitle = (props: HvDialogTitleProps) => {
   const { classes, cx } = useClasses(classesProp);
   const { fullScreen } = useDialogContext();
 
-  const icon = customIcon || (showIcon && iconVariant(variant));
+  const icon = customIcon || (showIcon && <HvStatusIcon variant={variant} />);
```

Comment:
there are regressions in other themes, that Chromatic isn't picking up

<img width="194" height="143" alt="image" src="https://github.com/user-attachments/assets/6b917961-71fd-4da7-a0e5-42e32027dadf" />

Reply:
@zettca this should now be fixed